Resources/Textures/Mobs/Species/Arachnid/organs.rsi/meta.json

```diff
@@ -0,0 +1,20 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "PixelTK",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+	{
+      "name": "eyeball-l"
+    },
+	{
+      "name": "eyeball-r"
+    },
+	{
+      "name": "tongue"
+    }
+  ]
+}
```

Comment:
Formatting needs fixing and copyright needs something more like a github user or discord handle.